code/modules/mob/living/carbon/life.dm

```diff
@@ -302,7 +298,13 @@
 
 //Fourth and final link in a breath chain
 /mob/living/carbon/proc/handle_breath_temperature(datum/gas_mixture/breath)
-	return
+	// The air you breath out should about match your body temperature
+	if(bodytemperature > breath.temperature)
+		// Raise the temperature of our breath to just below body temp
+		breath.temperature = bodytemperature - BREATH_TEMPERATURE_OFFSET
+	else
+		// The air is hotter than out body so we need to lower the air temp
+		breath.temperature = bodytemperature + BREATH_TEMPERATURE_OFFSET

```

Comment:
Is the offset necessary? Pretty sure air you breathe in quickly heats to exactly your internal temperature. Plus this creates edge cases where you could potentially breathe out a temperature that is a few degrees higher/lower than both the air and your body.

Reply:
This was done for a bit more real style as your breath is a touch lower.
I have no issue removing that.